tools/finish_release.py

```diff
@@ -125,12 +129,17 @@ def create_github_release(github_access_token, tempdir, version):
                                      release_notes,
                                      draft=True)
 
+    # SSH into CSS and sign executable, then retrieve
+    username = getpass.getuser()
```

Comment:
This makes the assumption that the local username and the username on the CSS are the same. This is probably fine, but I think we should call this out in any documentation about creating an account on the CSS.

Reply:
Agreed. I can add an optional argument for this use case.

Reply:
You can do that if you want, but I think it may also not be needed. I was mainly just noting this assumption and saying that we should probably at least document it somewhere.